src/Engine/Waypoint/Waypoint.cpp

```diff
@@ -44,3 +45,26 @@ Waypoint::Project(const FlatProjection &projection) noexcept
   flat_location_initialised = true;
 #endif
 }
+
+std::map<Waypoint::Type, tstring> Waypoint::TypeMap = {
```

Comment:
What could be a simple plain array, initialised at compile-time into `.rodata`, is now a runtime-initialised `std::map` with lots of heap allocations, with a complicated lookup procedure.